UnitsNet/UnitDefinitions/Pressure.json

```diff
@@ -272,8 +272,8 @@
     {
       "SingularName": "TonneForcePerSquareCentimeter",
       "PluralName": "TonnesForcePerSquareCentimeter",
-      "FromUnitToBaseFunc": "x*98066501.9960652",
-      "FromBaseToUnitFunc": "x*1.01971619222242E-08",
+      "FromUnitToBaseFunc": "x*9.80665e7",
+      "FromBaseToUnitFunc": "x/9.80665e7",
```

Comment:
Nice that you are making this consistent with the guideline!

Reply:
When the tests failed due to relative checks, I couldn't know if the conversion was wrong or the test, so I did both :)